src/rgw/const.py

```diff
@@ -86,9 +86,9 @@
 class RgwEndpoint(Enum):
     """Enum class to define rgw endpoints provided by hare."""
 
-    MOTR_PROFILE_FID = 'motr profile fid'
-    MOTR_HA_EP       = 'motr ha endpoint'
-    MOTR_CLIENT_EP   = 'motr my endpoint'
-    MOTR_PROCESS_FID = 'motr my fid'
+    profile_fid = 'motr profile fid'
```

Comment:
let them be in capital given constants

Reply:
Previously we read from sysconfig file and LHS constants were the keys.
Now in fetch-fids cmd, these are the new keys

Reply:
Done